src/Orleans/Statistics/RuntimeStatisticsGroup.cs

```diff
@@ -12,16 +12,16 @@ internal class RuntimeStatisticsGroup
     {
         private static readonly TraceLogger logger = TraceLogger.GetLogger("RuntimeStatisticsGroup", TraceLogger.LoggerType.Runtime);
 
-        private PerformanceCounter cpuCounter;
-        private PerformanceCounter availableMemoryCounter;
+        private PerformanceCounter cpuCounterPF;
```

Comment:
What does 'PF' suffix stand for? "PerformanceCounter"? Why did you need add the suffix to all the variable names?